packages/gatsby-source-shopify/CHANGELOG.md

```diff
@@ -1,850 +0,0 @@
-# Change Log
```

Comment:
The changelog shouldn't be deleted

Reply:
Whoops, good catch, adding the changelog back in.